src/components/customer-resource-show/customer-resource-show.css

```diff
@@ -19,3 +19,14 @@
 .detail-container-header {
   margin: 2rem 0;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default with you apply `display: flex` to any element. So i think removing line 25 would have the same effect.

Reply:
Thanks for your feedback Elrick! You are right, row is the default direction. Testing the modified code now. Will have it pushed up soon.